library/build.gradle

```diff
@@ -16,6 +16,13 @@ apply plugin: 'signing'
 version '3.2.2'
 group 'io.nlopez.smartlocation'
 
+ext {
+    sonatypeRepo = ""
```

Comment:
You should probably add a function here to check if the values have been set via cmdline or whatever, and fallback to "" if not. That way my publishing scripts for this library won´t fail, making my life easier :)